packages/remix-dev/compiler/compileBrowser.ts

```diff
@@ -73,7 +73,10 @@ const createEsbuildConfig = (
 
   let plugins: esbuild.Plugin[] = [
     deprecatedRemixPackagePlugin(options.onWarning),
-    cssFilePlugin(options),
+    cssFilePlugin({
+      mode: options.mode,
+      rootDir: config.rootDirectory,
```

Comment:
`rootDirectory`

Reply:
Yeah, got distracted and pushed before I finished lolol